### PR TITLE
text.trim, text.pad_left and text.pad_right

### PR DESCRIPTION
**Pull request by earncef**

### CODE REVIEW

File: stdlib/text.go [hunk 1]
```diff
@@ -440,6 +443,116 @@ func textSubstring(args ...objects.Object) (ret objects.Object, err error) {
...
+			}
+			return
+		}
+	}
+
+	sLen := len(s1)
+	padStrLen := len(s3)
+
+	if sLen >= i2 || padStrLen == 0 {
+		ret = &objects.String{Value: s1}
```
Comment (d5) [inline stdlib/text.go]: I think we can just return `args[0]`?

**Comment by d5** [comment]
Please update documentation too :)

File: stdlib/text.go (hunk 1 above)
Comment (earncef) [reply stdlib/text.go]: args[0] might not actually be a string.

Comment (d5) [reply stdlib/text.go]: You're right. Haha.

**Comment by earncef** [comment]
All done

**Pull request merged**